packages/app/src/cli/models/extensions/extension-instance.ts

```diff
@@ -149,8 +149,8 @@ export class ExtensionInstance<TConfiguration extends BaseConfigType = BaseConfi
     this.entrySourceFilePath = options.entryPath ?? ''
     this.directory = options.directory
     this.specification = options.specification
-    this.devUUID = `dev-${randomUUID()}`
     this.handle = this.buildHandle()
+    this.devUUID = `dev-${nonRandomUUID(this.handle)}`
```

Comment:
With this we ensure that the `devUUID` is always the same for the same extension. Allowing us to reload it if we need it.

Reply:
Noting that the handle can change (right?) so it's not 100% foolproof but it's close enough that it should be fine.

Reply:
It shouldn't change during `dev` though, which solves the issue of "reloading" the app.

In any case, this is mostly for ui-extensions, and we need to rethink why ui-extensions need a `devUUID` in the first place. Will probably do it in early january.